templates/generator.html

```diff
@@ -12,9 +12,13 @@
     },
 
     data: {
-        start: "Start Article",
-        end: "End Article",
+        article: "Article",
+    },
 
+    methods: {
+        async generateRndPrompt() {
+            [ this.article ] = await this.$refs.pg.generatePrompt();
```

Comment:
I think the two-way binding was definitely a little complicated, but if we want to make this component completely untied from the input, an emitted event might be better.

Reply:
A few reasons:
1. Following the single responsibility principle, we should make our components as small and simple as possible, i.e. do just 1 job. In the case of the generator, the only required parameter is the difficulty. We should keep all other templating/ui outside. Hence, why I removed the two-way binding on the start and end articles.
2. Considered using emit but eventually decided against it because with emit there's no way to bind contextually blindly i.e. how would you intercept what emit event is for which input group without coding them separately in the generator component.